contracts/gateway.yaml

```diff
@@ -3972,11 +4086,102 @@ components:
           properties:
             purpose:
               $ref: '#/components/schemas/PatientAuthPurpose'
-            modes: 
+            modes:
               type: array
               items:
                 $ref: '#/components/schemas/AuthenticationMode'
-        error: 
+        error:
+          $ref: '#/components/schemas/Error'
+        resp:
+          $ref: '#/components/schemas/RequestReference'
+    PatientShareRequest:
+      type: object
+      required:
+        - requestId
+        - timestamp
+        - patient
+        - hipDetails
+      properties:
+        requestId:
+          type: string
+          format: uuid
+          description: a nonce, unique for each HTTP request
+          example: 5f7a535d-a3fd-416b-b069-c97d021fbacd
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        hipDetails:
+          type: object
+          properties:
+            hipId:
+              type: string
+              example: "100005"
+            code:
+              type: string
+        patient:
+          type: object
+          properties:
+            healthId:
+              type: string
+              example: <username>@<suffix>
+            name:
+              type: string
+              example: "Jane Doe"
+            gender:
+              $ref: '#/components/schemas/PatientGender'
+            address:
+              type: string
+            state:
+              type: string
+            district:
+              type: string
+            yearOfBirth:
+              type: integer
+              example: 2000
+            dateOfBirth:
+              type: string
+              example: "1972-02-29"
+              description: date of birth in YYYY-MM-DD format.
+            monthOfBirth:
+              type: string
+              example: "1972-02-29"
```

Comment:
why is month-of-birth a date type? Why not a simple integer?

Reply:
done